### ISSUE TITLE
[Request] Video Tutorials 

### ISSUE THREAD

**Issue by pmfcdb** [xenko3d/xenko#257]
Hi, I check your youtube channel and there isn't any video tutorials. And I think it is a good way to get more people into this 3d engine.  Like this:

https://unity3d.com/learn/tutorials/s/roll-ball-tutorial
or
https://academy.unrealengine.com
or
https://www.cryengine.com/tutorials
Thks,
PB

**Comment by tebjan** [comment]
hi @pmfcdb you are absolutely right. only thing is, xenko is a community driven open source project. there is no company behind it that has money to pay someone to do that, which is a huge job. so we need people from from community to step forward and make something like that.

the written documentation is quite good though, and if you can read english and you seriously want to work on a project, the written documentation has you covered: https://doc.xenko.com/latest/en/manual/index.html

so who has time to start some?

**Comment by SimantoR** [comment]
Is there a donation system for the project? Maybe we could do a proper donation system for the development of Xenko?

**Comment by profan** [comment]
@SimantoR https://www.patreon.com/xenko/overview this exists, which I imagine is also exactly what its supposed to be

**Comment by herocrab** [comment]
We have a series of nice videos from @Aggror now. Can this be closed?

**Comment by Eideren** [comment]
I don't think this issue should ever be closed given that there probably will never be enough tutorials on the engine. I think we should leave this open as a place for users to discuss this subject.

**Comment by Rogardo** [comment]
I recently find the stride tutorials for beginners on Youtube. 
Is there somewhere, a tutorial (text or video) how to work with sprites or general in 2D?
